packages/grafana-ui/src/components/QueryField/QueryField.test.tsx

```diff
@@ -1,30 +1,43 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import { QueryField } from './QueryField';
+import { UnThemedQueryField } from './QueryField';
 import { Editor } from 'slate';
+import { createTheme } from '@grafana/data';
```

Comment:
One of our goals in the frontend is to update all our enzyme tests to use RTL, at first impression this test looks like it might take a long time, but is it possible to add this migration in another issue? or maybe this PR 🙈

Reply:
I think we should tackle that in another PR to keep this small and get it out quickly